PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD003.xml

```diff
@@ -0,0 +1,231 @@
+                                <decoNote xml:id="b1">The boards of the manuscript are missing. The textblock is held together by the sewing.</decoNote> 
+                             <decoNote xml:id="b2" type="SewingStations">4</decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    
+                    <history>
+                        <origin>
+                          <origDate notBefore="1400" notAfter="1500" evidence="lettering">
+                                1400–1500 </origDate>.
+                        </origin>
+                        <provenance>No colophon notes are present in the manuscript, but the owner 
+                            <persName ref="PRS14417MarigetaDamseEndalew" role="owner"/>
+                            who lives in<placeName ref="LOC2386DabraT">
+                                Dabra Tābor ʾIyasus parish</placeName> confirmed it as the manuscript is transmitted in his family in the 6th generation.
+                            <persName ref="PRS14417MarigetaDamseEndalew" role="scribe"/>. 
+                            </provenance>
+                    </history> 
+                    
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
+                href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+                <xi:fallback>
+                    <p>Definitions of prefixes used.</p>
+                </xi:fallback>
+            </xi:include>
+        </encodingDesc>
+        <profileDesc>
+            <particDesc>
+                <listPerson>
+                    <person>
+                        <persName ref="PRS14417MarigetaDamseEndalew" role="owner">Marigetā Damḍa ʾƎndālaw</persName>
+                    </person>
+                </listPerson>
+            </particDesc>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Geez</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="AY" when="2024-07-25">created the record</change>
+            <change who="AY" when="2024-08-02">added incipit and explicit</change>
+            <change who="AY" when="2024-08-05">added quire structure, foliation, measurement, layout and binding</change>
+            <change who="AY" when="2024-08-07">added binding description</change>
+            <change who="AY" when="2024-08-07">added origin history description</change>
+            <change who="AY" when="2024-09-16">added date of the manuscript.</change>
```

Comment:
I would not recommend to be that much precise with the change elements in future. "created the record" is absolutely sufficient, if you are the only one, who worked on this record within a relatively short time.